features/dd-sdk-android-trace/src/main/java/com/datadog/opentracing/propagation/DatadogHttpCodec.java

```diff
@@ -31,7 +31,9 @@ class DatadogHttpCodec {
 
     public static final String OT_BAGGAGE_PREFIX = "ot-baggage-";
     public static final String LEAST_SIGNIFICANT_TRACE_ID_KEY = "x-datadog-trace-id";
-    public static final String MOST_SIGNIFICANT_TRACE_ID_KEY = "_dd.p.tid";
+    public static final String MOST_SIGNIFICANT_TRACE_ID_TAG_KEY = "_dd.p.tid";
+    public static final String RUM_SESSION_ID_TAG_KEY = "_dd.p.rsid";
+    public static final String RUM_SESSION_ID_KEY = "session_id";
```

Comment:
won't this key have a risk to overlap with customer-set key? maybe we should prefix it with Datadog/RUM?

Reply:
This key is a tag we set ourselves in the Android Tracer on the span, and the customer doesn't have access to that span before the generation of the headers.

Reply:
Won't APM backend process this tag as well in the `SpanEvent` received? Or it is used only for the propagation?

Reply:
It's only used for propagation